turborepo-tests/integration/tests/affected.t

```diff
@@ -60,6 +63,36 @@ Do the same thing with the `query` command
 Remove the new file
   $ rm apps/my-app/new.js
 
+Add a file in `util`
+  $ echo "hello world" > packages/util/new.js
+
+Validate that both `my-app` and `util` are affected
+  $ ${TURBO} query "query { affectedPackages { items { name reason { __typename } } } }"
```

Comment:
What's with the `__typename` thing? Is that going to stay? Can it just be `reason` (and the response also not have the value nested?)